nixos/modules/services/networking/mosquitto.nix

```diff
@@ -0,0 +1,220 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.mosquitto;
+
+  listenerConf = if cfg.ssl.enable then ''
```

Comment:
There's also `optionalString` which encodes the `if cond then str else ""` pattern.